src/Microsoft.VisualStudio.Web.CodeGeneration/Properties/AssemblyInfo.cs

```diff
@@ -1,11 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Reflection;
-using System.Resources;
-
-[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
We still want to keep this attribute on our assemblies. AFAIK Microsoft.NET.Sdk does not yet generate this.